test/uk/gov/hmrc/agentservicesaccount/config/ExternalUrlsSpec.scala

```diff
@@ -35,7 +35,11 @@ class ExternalUrlsSpec extends UnitSpec with ResettingMockitoSugar {
   val completeGgSignInUrl = s"$companyAuthFrontendExternalBaseUrl$ggSignInPath?continue=${urlEncode(externalUrl + routes.AgentServicesController.root())}"
   val completeGgSignOutUrl = s"$companyAuthFrontendExternalBaseUrl$ggSignOutPath?continue=${urlEncode(ggSignOutContinueUrl)}"
 
-  val configuration = resettingMock[Configuration]
+  val mappingExternalUrl = "http://www.example.com/foo"
```

Comment:
`mappingExternalUrl` is a base URL so won't have a path - `val mappingExternalUrl = "http://www.example.com"` would be more realistic